certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk3226/CertificateModelFactoryFK3226.java

```diff
@@ -150,8 +161,137 @@ public class CertificateModelFactoryFK3226 implements CertificateModelFactory {
   public static final SchematronPath SCHEMATRON_PATH = new SchematronPath(
       "fk3226/schematron/lunsp.v1.sch");
 
-  public static final String PDF_FK_3226_PDF = "fk3226/pdf/fk3226_v1.pdf";
-  public static final String PDF_NO_ADDRESS_FK_7472_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf";
+  public static final String PDF_FK_3226_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf"; //TODO: update when new template has been provided
+  public static final String PDF_NO_ADDRESS_FK_3226_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf";
+  public static final Mcid PDF_MCID = new Mcid(100);
```

Comment:
From what I can tell this mcid works, but I'm not 100% sure.

Reply:
We will test it and modify it later if necessary.